SimGeneral/MixingModule/python/digitizers_cfi.py

```diff
@@ -101,6 +101,9 @@
 
 theDigitizersValid = cms.PSet(theDigitizers)
 theDigitizers.mergedtruth.select.signalOnlyTP = True
+from Configuration.Eras.Modifier_run3_ecalclustering_cff import run3_ecalclustering
+run3_ecalclustering.toModify( theDigitizersValid,
+                       calotruth = cms.PSet( caloParticles ) )
 
 phase2_hgcal.toModify( theDigitizersValid,
```

Comment:
These could be combined
```python
(run3_ecalclustering | phase2_hgcal).toModify(theDigitizersValid, ...)
```
since they apply the same modification.